### ISSUE TITLE
Use sklean.shuffle

### ISSUE THREAD

**Issue by marcelsan** [wmr1/redes_neurais#5]
from sklearn.utils import shuffle
